src/modules/search/search-intent.js

```diff
@@ -12,25 +12,26 @@ const dictionary = require('./dictionary');
  *      - 'What is <item>?'
  */
 module.exports = app => {
-  app.customSlot('Item', Object.keys(dictionary.items));
-
-  const utterances = [
-    'Search for {item:Item}',
-    'What is {item:Item}',
-    'Describe me {item:Item}'
-  ];
-
-  app.intent('SearchIntent', utterances, (slots, attrs, data, done) => {
+  app.intent('SearchIntent', (slots, attrs, data, done) => {
     if (!slots.item) {
-      done('I did not understand your search item');
+      done(
+        app.t('text_error1')
+      );
       return;
     }
 
-    dictionary.search(slots.item).then(result => {
+    const locale = data.request.locale || app.i18next.language || 'en-US';
```

Comment:
this line is missing tests coverage for all possible branches
run `npm test` and add tests to cover all branches